source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizer.xcodeproj/project.pbxproj

```diff
@@ -440,6 +448,10 @@
 		F4D33E341F045B6E00941E44 /* Jsons */ = {
 			isa = PBXGroup;
 			children = (
+				6BE0389422DE46C1005413DE /* Agenda.json */,
+				6BE0389322DE46C1005413DE /* ExpenseReport.json */,
+				6BE0389522DE46C1005413DE /* FlightDetails.json */,
+				6BE0389122DE465B005413DE /* Column.Fallback.json */,
```

Comment:
>Column.Fallback.json [](start = 32, length = 20)

I added another test file in my change: samples/v1.2/Tests/ContainerDoubleFallback.json. Should that show up here?

Reply:
thanks for pointing it out, I just added it, and attached a screen shot to the PR. 

---
In reply to: [304554842](https://github.com/microsoft/AdaptiveCards/pull/3238#discussion_r304554842) [](ancestors = 304554842)